packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -66,6 +66,7 @@ export class ModelScene extends Scene {
   public element: ModelViewerElement;
   public canvas: HTMLCanvasElement;
   public annotationRenderer = new CSS2DRenderer();
+  public effectsRenderer: EffectsRendererInterface|null = null;
```

Comment:
Starting a thread here for performance and testing:

OS's tested: Windows, WSL2/vmware, macOS, IOS | Effects work identically on all systems
Devices: 
- Mac Air 2013 (1.3 ghz dual core i5, intel HD Graphics 5000, 4gb ram)
- Gigabyte G5 (i5-10500H, intel UHD 630, 16gb ram)
- iPhone 11

Browsers: Chrome, Firefox, Safari | Effects work identically on all browsers

I haven't noticed a significant performance reduction with any of the above device/browser configurations.

Reply:
Do you have a cheap old Android phone to test? That's where performance tends to make a big difference.